### DISCUSSION TITLE
Chiling-Next希望从各位争取意见。

### DISCUSSION THREAD

**Discussion by MFn233** [Meloong-Git/PCL#4661] [自定义主页]
各位可以提出老chiling主页出现的问题，我将一一解决。由于月末没流量，请各位去原库查看老chiling主页，敬请谅解。

**Comment by LTCatt** [comment]
有网址么……（

**Reply by MFn233** [reply]
https://github.com/MFn233/PCL-Mainpage-ChiLing
我网速怎么突然变快

**Reply by LTCatt** [reply]
![image](https://github.com/user-attachments/assets/c26f92fc-fa61-41a5-98dc-706ef2d0498b)

**Reply by Ad-closeNN** [reply]
[https://mfn233.github.io/PCL-Mainpage-ChiLing/Custom.xaml](https://mfn233.github.io/PCL-Mainpage-ChiLing/Custom.xaml)
![image](https://github.com/user-attachments/assets/99493778-a739-4f4e-a937-6f5d154d88cf)

**Reply by MFn233** [reply]
> [https://mfn233.github.io/PCL-Mainpage-ChiLing/Custom.xaml](https://mfn233.github.io/PCL-Mainpage-ChiLing/Custom.xaml)
> ![image](https://github.com/user-attachments/assets/99493778-a739-4f4e-a937-6f5d154d88cf)
> 

第一个 card 的草没加载出来，等到后续我会更换图床之类的

**Comment by hejiehao** [comment]
NukkitX 的地址是不是放错了（

**Comment by wkea** [comment]
整体不错

**Comment by LTCatt** [comment]
感觉可以考虑把这个和简单主页合并一下……

**Reply by MFn233** [reply]
抱歉龙猫，不会考虑这件事。

**Reply by MFn233** [reply]
简单主页需要简洁快速，而不是过大的文件体积、过多炫技内容、过多自定义 style

**Reply by LTCatt** [reply]
我个人觉得没有必要把相同定位的东西拆分到两个不同的预设里，如果不打算合并的话那就不加进去了……（

**Comment by LTCatt** [comment]
@MFn233 简单主页的 GitCode 链接现已无法访问

**Reply by MFn233** [reply]
极有可能是raw引用的行为被拒绝了……
不知道有没有gitcode pages或者国内其它pages服务可以用

**Reply by MFn233** [reply]
可能会将主页迁移到主页广场OSS，感谢提醒……

**Reply by MFn233** [reply]
@LTCatt 已经修改链接到
https://pclhomeplazaoss.lingyunawa.top:26994/d/Homepages/MFn233/Custom.xaml

**Reply by LTCatt** [reply]
@MFn233  这个链接也已经无法访问

**Reply by CreeperIsASpy** [reply]
@LTCatt HPoP的上游服务商 MK 最近好像出了点问题。
